frontend/reducers/currentSection/currentSection_reducer.jsx

```diff
@@ -0,0 +1,7 @@
+export default function currentSectionReducer (state="Home", action) {
```

Comment:
@jasonjunglee Let's use ES6 for everything: `export default const currentSectionReducer`